bang/migrations/0056_chibi_mover.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+from django.utils import timezone
+import magi.utils
+
+def move_card_chibis(apps, schema_editor):
+    Card = apps.get_model('bang', 'Card')
+    Costume = apps.get_model('bang', 'Costume')
+    NewChibi = apps.get_model('bang', 'Chibi')
+
+    for card in Card.objects.all().select_related('associated_costume').order_by('id'):
+        chibis = card.chibis.all()
+        if not chibis:
+            continue
+
+        if hasattr(card, 'associated_costume'):
+            costume = card.associated_costume
+        else:
+            costume = Costume()
+            costume.owner_id = card.owner_id
+            costume.i_costume_type = 0 # get_i('costume_type', 'live')
+            costume.card = card
+            costume.member = card.member
+            costume.save()
+
+        for the_image in card.chibis.all():
+            new_chibi = NewChibi()
+            new_chibi.image = the_image.image
```

Comment:
Since the ones we currently have in db are compressed, you can add the following to avoid re-calling TinyPNG to compress them again.

```python
new_chibi._original_image = the image.image
```

Reply:
It's also worth noting that we won't have any way to know which ones have been compressed already or not if we decide to re-upload them to provide un-compressed ones, but I don't think we have any plan to do that.

Reply:
fixed this in 55a7c48!